src/components/Header/Header.tsx

```diff
@@ -0,0 +1,143 @@
+import styles from './header.module.css';
+
+function Header() {
+    return (
+        <header className={styles.header}>
+            <div className={styles.headerContainer}>
+                <a href="./">
+                    <svg className={styles.headerLogoIcon} xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 40 40">
```

Comment:
Transfer all SVG images to separate files.

Reply:
Fixed